py3status/modules/arch_updates.py

```diff
@@ -28,20 +32,34 @@
 aur
 {'full_text': 'UPD: 15/4'}
 """
+import os
+import time
 
 STRING_NOT_INSTALLED = "{} not installed"
+CACHE_KEY_TEXT = "arch_updates_text"
+CACHE_KEY_TIMESTAMP = "arch_updates_timestamp"
 
 
 class Py3status:
     """ """
 
     # available configuration parameters
-    cache_timeout = 600
     format = None
     hide_if_zero = False
+    pacman_log_location = "/var/log/pacman.log"
```

Comment:
(Opinion) I'd like the module to just parse /etc/pacman.conf and grab the log location from there instead.